modules/microshift-default-settings.adoc

```diff
@@ -19,29 +18,60 @@ $ microshift show-config
 .Default values example output in YAML form
 [source,yaml]
 ----
+apiServer:
+  advertiseAddress: 10.44.0.0/32 # <1>
+  auditLog:
+    maxFileAge: 10
+    maxFileSize: 200
+    maxFiles: 10
+    profile: Default
+  namedCertificates:
+    - certPath: ""
+      keyPath: ""
+      names:
+        - ""
+  subjectAltNames: [] # <2>
+debugging:
+  logLevel: "Normal" # <3>
 dns:
-  baseDomain: microshift.example.com <1>
+  baseDomain: microshift.example.com <4>
+etcd:
+  memoryLimitMB: 0 # <5>
+ingress:
+  listenAddress:
+    - ""
+  ports:
+    http: 80
+    https: 443
+  routeAdmissionPolicy:
+    namespaceOwnership: InterNamespaceAllowed # <6>
+  status: Managed # <7>
+manifests: # <8>
+  kustomizePaths:
+    - /usr/lib/microshift/manifests
+    - /usr/lib/microshift/manifests.d/*
+    - /etc/microshift/manifests
+    - /etc/microshift/manifests.d/*
 network:
   clusterNetwork:
-    - 10.42.0.0/16 <2>
+    - 10.42.0.0/16 # <9>
   serviceNetwork:
-    - 10.43.0.0/16 <3>
-  serviceNodePortRange: 30000-32767 <4>
+    - 10.43.0.0/16 # <10>
+  serviceNodePortRange: 30000-32767 # <11>
 node:
-  hostnameOverride: "" <5>
-  nodeIP: "" <6>
-apiServer:
-  advertiseAddress: 10.44.0.0/32 <7>
-  subjectAltNames: [] <8>
-debugging:
-  logLevel: "Normal" <9>
+  hostnameOverride: "" # <12>
+  nodeIP: "" # <13>
 ----
-<1> Base domain of the cluster. All managed DNS records will be subdomains of this base.
-<2> A block of IP addresses from which Pod IP addresses are allocated.
-<3> A block of virtual IP addresses for Kubernetes services.
-<4> The port range allowed for Kubernetes services of type NodePort.
-<5> The name of the node. The default value is the hostname.
-<6> The IP address of the node. The default value is the IP address of the default route.
-<7> A string that specifies the IP address from which the API server is advertised to members of the cluster. The default value is calculated based on the address of the service network.
-<8> Subject Alternative Names for API server certificates.
-<9> Log verbosity. Valid values for this field are `Normal`, `Debug`, `Trace`, or `TraceAll`.
\ No newline at end of file
+<1> A string that specifies the IP address from which the API server is advertised to members of the cluster. The default value is calculated based on the address of the service network.
+<2> Subject Alternative Names for API server certificates.
+<3> Log verbosity. Valid values for this field are `Normal`, `Debug`, `Trace`, or `TraceAll`.
+<4> By default, `etcd` uses as much memory as needed to handle the load on the system. However, in memory constrained systems, it may be preferred or necessary to limit the amount of memory `etcd` can to use at a given time.
```

Comment:
```suggestion
<4> By default, `etcd` uses as much memory as needed to handle the load on the system. However, in memory constrained systems, it might be preferred or necessary to limit the amount of memory `etcd` can to use at a given time.
```